tests/lib/boot.sh

```diff
@@ -11,12 +12,16 @@ bootenv() {
     if [ $# -eq 0 ]; then
         if command -v "$GRUB_EDITENV" >/dev/null; then
             "$GRUB_EDITENV" list
+        elif [ -s "$GRUBENV_FILE" ]; then
+            cat "$GRUBENV_FILE"
         else
             fw_printenv
         fi
     else
         if command -v "$GRUB_EDITENV" >/dev/null; then
             "$GRUB_EDITENV" list | grep "^$1"
+        elif [ -s "$GRUBENV_FILE" ]; then
+            grep "^$1" "$GRUBENV_FILE"
```

Comment:
Wonder whether it should be `grub "^$1=" "$GRUBENV_FILE"`, but looking how other query options fail to do it this way, I suppose this is fine.